mdm/service.go

```diff
@@ -14,10 +14,12 @@ type Service interface {
 type Middleware func(Service) Service
 
 const (
-	ConnectTopic      = "mdm.Connect"
-	AuthenticateTopic = "mdm.Authenticate"
-	TokenUpdateTopic  = "mdm.TokenUpdate"
-	CheckoutTopic     = "mdm.CheckOut"
+	ConnectTopic           = "mdm.Connect"
+	AuthenticateTopic      = "mdm.Authenticate"
+	TokenUpdateTopic       = "mdm.TokenUpdate"
+	CheckoutTopic          = "mdm.CheckOut"
+	GetBootstrapTokenTopic = "mdm.GetBootstrapToken"
+	SetBootstrapTokenTopic = "mdm.SetBootstrapToken"
 )
 
```

Comment:
Below here, line 31 in the `MDMService` is where we'll couple our device service to the check-in service (using a simplified API, hopefully).